content/04.network_applications.md

```diff
@@ -34,6 +34,7 @@ $A$ is constructed based on the knowledge graph's structure itself and collapses
 Following construction, $A$ is decomposed into the following three parts: a square matrix $Σ$ and a set of two small rectangular matrices $U$ and $V^{T}$.
 Values within $Σ$ are called singular values, which are akin to eigenvalues [@doi:10.1007/BF02288367].
 Each row in $U$ and each column in $V^{T}$ represents nodes within a low dimensional space [@doi:10.1007/BF02288367; @doi:10/bm8dnf].
+This low dimensional representation loses the concept of a knowledge graph as only nodes are explicitly represented in this context. 
```

Comment:
Do you want to say that it loses the _concept_ or that it no longer meets the definition? In this case, is there some position thing you can say about the representation? What do you get for giving up the knowledge graph?